cmd/play/main.go

```diff
@@ -13,7 +14,11 @@ func main() {
 		SizeLimit:    10000,
 	}
 
-	r := s.SetupRouter()
+	gin.SetMode("release")
+	gr := gin.New()
+	gr.Use(gin.Logger(), gin.Recovery())
```

Comment:
Shouldnt this be part of the `SetupRouter` ?

Reply:
Wanted different logging levels for prod and test, could always just pass a boolean for prod?